.github/workflows/windows_release_build.yml

```diff
@@ -32,10 +32,10 @@ jobs:
       - name: install rust nightly 1.77.0
         run: rustup install nightly-2024-02-03
 
-      - name: set up llvm 16
+      - name: set up llvm 18
         run: |
-          curl.exe -f -L -O -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" https://github.com/roc-lang/llvm-package-windows/releases/download/v16.0.6/LLVM-16.0.6-win64.7z
-          7z x LLVM-16.0.6-win64.7z -oC:\LLVM-16.0.6-win64
+          curl.exe -f -L -O -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" https://github.com/roc-lang/llvm-package-windows/releases/download/v18.1.8/LLVM-18.1.8-win64.7z
+          7z x LLVM-18.1.8-win64.7z -oC:\LLVM-18.1.8-win64
```

Comment:
We need a new release for Windows LLVM in https://github.com/roc-lang/llvm-package-windows/releases/